### PR TITLE
add juicefs chart

### PR DESCRIPTION
**Pull request by zwwhdls**

### CODE REVIEW

File: src/stable/juicefs/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+type: application
+version: 0.6.1
+appVersion: 0.10.6
+kubeVersion: ">=1.14.0-0"
+home: https://github.com/juicedata/juicefs
+sources:
+  - https://github.com/juicedata/juicefs
+keywords:
+  - juicefs
+  - csi
```
Comment (zheng1) [inline src/stable/juicefs/Chart.yaml:13]: Is it possible to add the maintainers field here? For showing it in kubesphere

Comment (zwwhdls) [reply src/stable/juicefs/Chart.yaml:17]: done

File: src/stable/juicefs/values.yaml [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# -- Overrides the chart's computed name
+# nameOverride: ""
+# -- Overrides the chart's computed fullname
+# fullnameOverride: ""
+
+image:
+  repository: juicedata/juicefs
```
[the remark this answers is not in the document]
Comment (zwwhdls) [reply src/stable/juicefs/values.yaml]: Sorry for this mistake, fix it already.

**Pull request merged**